draft-dekater-scion-controlplane.md

```diff
@@ -1070,6 +1072,16 @@ On code-level and in Protobuf message format, extensions are specified as follow
 **Note:** SCION also supports so-called "detachable extensions". The detachable extension itself is part of a PCB's unsigned extensions, but a cryptographic hash of the detachable extension data is added to the signed extensions. Thus, a PCB with a detachable extension can be signed and verified without actually including the detachable extension in the signature. This prevents a possible processing overhead caused by large cryptographically-protected extensions.
 
 
+### Configuration
+
+For the purpose of constructing and propagating path segments, an AS control service must be configured with links to neighboring ASes. Such information may be conveyed to the control service in an out of band fashion (e.g in a configuration file). For each link, these values must be configured:
+
+- Local interface ID
+- Neighbor type (core, parent, child, peer), depending on link type (see [](#paths-links)). Link type depends on mutual agreements between the organizations operating the ASes at each end of each link.
+- Neighbor ISD-AS number
+- Neighbor interface underlay address
+
```

Comment:
@matzf could you double check that the list is correct?

Reply:
I removed the underlay addresses from the list. Also used the same intro paragraph as in the DP draft.

Reply:
:+1: